nvflare/app_common/widgets/streaming.py

```diff
@@ -310,3 +281,38 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
         elif event_type == EventType.END_RUN:
             self._end = True
             self.finalize(fl_ctx)
+
+
+class LogSender(Widget, logging.StreamHandler):
+    def __init__(self, log_level=""):
+        """
+        LogSender for sending the logging record to the FL server.
```

Comment:
This should be in the same line as """

So -> `"""LogSender for sending the logging record to the FL server.`

Reply:
If we are following this style guide: https://google.github.io/styleguide/pyguide.html